build/Microsoft.DotNet.Cli.Prepare.targets

```diff
@@ -96,12 +96,13 @@
       <BundleExtension Condition=" '$(OSName)' == 'osx' ">$(InstallerExtension)</BundleExtension>
       <BundleExtension Condition=" '$(OSName)' == 'ubuntu' ">$(InstallerExtension)</BundleExtension>
 
-      <ProductMonikerRid Condition=" '$(Rid)' == 'ubuntu.16.04-x64' Or 
-                                     '$(Rid)' == 'ubuntu.16.10-x64' Or 
-                                     '$(Rid)' == 'fedora.23-x64' Or 
-                                     '$(Rid)' == 'fedora.24-x64' Or 
-                                     '$(Rid)' == 'opensuse.13.2-x64' Or 
-                                     '$(Rid)' == 'opensuse.42.1-x64' ">$(Rid)</ProductMonikerRid>
+      <ProductMonikerRid Condition=" '$(Rid)' == 'ubuntu.16.04-x64' Or
+                                     '$(Rid)' == 'ubuntu.18.04-x64' Or
+                                     '$(Rid)' == 'fedora.24-x64' Or
```

Comment:
Should Fedora 24 go away here too?

Reply:
@janvorli 
I would like to keep the ability to build Fedora 24 and openSUSE 42.1 - we will not build either officially. Moreover, if we remove Fedora 24, CI will most likely break.

Reply:
Ah, sorry for the misunderstanding then. I've thought that it was to be removed completely.